tools/depends/README

```diff
@@ -1,35 +1,49 @@
-Temporary readme:
-
 Examples:
 
-OSX:
-  x64:
-    ./configure --host=x86_64-apple-darwin # (defaults chosen)
-  x86:
+#- Darwin
+#----------------------------------------------------------------------------
+
+OSX (i386):
     ./configure --host=i386-apple-darwin
 
-IOS:
-  ./configure --host=arm-apple-darwin # (defaults chosen)
-  ./configure --host=arm-apple-darwin --with-sdk=4.3 --prefix=home/foo/xbmc-deps
+OSX (x86_64):
+    ./configure --host=x86_64-apple-darwin
+
+IOS (armv7):
+    ./configure --host=arm-apple-darwin
+
+IOS (arm64):
+    ./configure --host=arm-apple-darwin --with-cpu=arm64
+
+TVOS:
+    ./configure --host=arm-apple-darwin --with-cpu=arm64 --with-platform=tvos
```

Comment:
Not yet - but i will change configure.ac to make it implicit in case its not given on the configure line...